SPECS/lua/lua.spec

```diff
@@ -31,14 +31,14 @@ Source3:        http://www.lua.org/tests/lua-%{test_version}-tests.tar.gz
 Source4:        luaconf.h
 Patch0:         %{name}-5.4.0-beta-autotoolize.patch
 Patch1:         %{name}-5.4.6-idsize.patch
-#Patch2:        %%{name}-5.3.0-luac-shared-link-fix.patch
 Patch3:         %{name}-5.2.2-configure-linux.patch
 Patch4:         %{name}-5.3.0-configure-compat-module.patch
 %if 0%{?bootstrap}
 Patch5:         %{name}-5.3.0-autotoolize.patch
 Patch6:         %{name}-5.3.5-luac-shared-link-fix.patch
 %endif
 # https://www.lua.org/bugs.html
+Patch7:         %{name}-5.4.6-CVE-2020-24371.patch
```

Comment:
This spec file does not leverage `%autosetup` (which applies the patches automatically), so we actually would want to use a `%patch -P 7 ...` directive (e.g line 94) to actually apply the patch :)

Reply:
I thought that was too easy. I'll see about getting the patch *actually* applied.